ee/clickhouse/queries/test/test_breakdown_props.py

```diff
@@ -113,3 +116,44 @@ def test_breakdown_person_props_with_entity_filter(self):
                 )
                 res = get_breakdown_prop_values(filter, Entity(entity_params[0]), "count(*)", self.team.pk, 5)
                 self.assertEqual(res, ["test"])
+
+    @snapshot_clickhouse_queries
+    def test_breakdown_group_props(self):
+        GroupTypeMapping.objects.create(team=self.team, group_type="organization", group_type_index=0)
+        GroupTypeMapping.objects.create(team=self.team, group_type="company", group_type_index=1)
+
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:5", properties={"industry": "finance"})
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:6", properties={"industry": "technology"})
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:7", properties={"industry": "finance"})
+        create_group(
+            team_id=self.team.pk, group_type_index=0, group_key="org:8", properties={"industry": "another", "out": 1}
+        )
+        create_group(
+            team_id=self.team.pk, group_type_index=1, group_key="company:10", properties={"industry": "foobar"}
+        )
+
+        for org_index in range(5, 9):
```

Comment:
I like how many scenarios this test covers.

I'd like to have another event with group_typ_index=1 to test the case where `group_type_index=1`, and event for this group **exists**, and is not in breakdown.

```suggestion
        for org_index in range(5, 11):
```

and

```
create_group(
            team_id=self.team.pk, group_type_index=1, group_key="org:8", properties={"industry": "foobar"}
        )
```

i.e. `group_key` is same across group types, but indexes are different, so they should be treated completely differently.